publiq-ui/pub-card.stories.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Story, Canvas, ArgsTable } from '@storybook/addon-docs/blocks';
+import PubCard from './pub-card.vue';
+import PubCardText from './pub-card-text.vue';
+
+<Meta title="Components/pub-card" component={PubCard} />
+
+export const TemplateCard = (args, { argTypes }) => {
+  return {
+    props: Object.keys(argTypes),
+    components: { PubCard },
+    template: '<pub-card v-bind="$props">Card content</pub-card>',
+  };
+};
+
+export const TemplateCardWithCardText = (args, { argTypes }) => {
+  return {
+    props: Object.keys(argTypes),
+    components: { PubCard, PubCardText },
+    template:
+      '<pub-card v-bind="$props">Card content <pub-card-text v-bind="$props">Card footer text</pub-card-text></pub-card>',
+  };
+};
+
+## Default
+
+<Canvas>
+  <Story name="Default">{TemplateCard.bind({})}</Story>
+</Canvas>
+
+```javascript
+<pub-card>Card content</pub-card>
+```
+
+## With card text
+
+<Canvas>
+  <Story name="WithText">{TemplateCardWithCardText.bind({})}</Story>
+</Canvas>
+
+```javascript
+<pub-card>
+  Card content
+  <pub-card-text>Card footer text</pub-card-text>
+</pub-card>
```

Comment:
What's the point of the `pub-card-text` / `b-card-text` exactly?

Reply:
It says "Card _footer_ text" (emphasis mine), but I see no difference with just regular text content inside `pub-card`?

Reply:
With sub-component <b-card-text>, paragraph text can be added to the card. The last <b-card-text> in the card body will have its bottom margin automatically removed (via CSS). Text within <b-card-text> can also be styled with the standard HTML tags.

Reply:
![Schermafbeelding 2020-09-11 om 13 29 32](https://user-images.githubusercontent.com/66943525/92920628-dd814480-f432-11ea-8cca-b856a1002cfd.png)

Reply:
So it are standard <p> tags I think only the last p tags margin is automatically  taken care of

Reply:
Aha I see! Sounds good. Maybe we could clarify the story then by changing the example to something like:
```
<pub-card>
  <pub-card-text>Paragraph 1</pub-card-text>
  <pub-card-text>Paragraph 2</pub-card-text>
  <pub-card-text>Paragraph 3 (will have its bottom margin removed)</pub-card-text>
</pub-card>
```

Reply:
Also maybe `pub-card-paragraph` would be clearer? 🤔 But more verbose. Just throwing the idea out there.

Reply:
changes made